Framework/DataHandling/src/LoadAscii2.cpp

```diff
@@ -514,13 +514,12 @@ void LoadAscii2::newSpectra() {
           m_curSpectra->setPointStandardDeviations(std::move(m_curDx));
         m_spectra.push_back(*m_curSpectra);
       }
-      delete m_curSpectra;
-      m_curSpectra = nullptr;
+      m_curSpectra.reset();
     }
 
-    m_curSpectra =
-        new DataObjects::Histogram1D(HistogramData::Histogram::XMode::Points,
-                                     HistogramData::Histogram::YMode::Counts);
+    m_curSpectra = Kernel::make_unique<DataObjects::Histogram1D>(
```

Comment:
Now that we require C++14, may we now use `std::make_unique`?

Reply:
It does look like all of our compiles now support this so I think we're good.

Reply:
Should we announce that people should now prefer `std::make_unique` of `Kernel` and create a maintenance task to replace everything?

Reply:
Yes, we need to announce this to more people. Since it is the first breaking change with GCC 4.8, we also need some discussion of changes in how we build on RHEL 7.

@VickieLynch @peterfpeterson